localstack/services/awslambda/lambda_api.py

```diff
@@ -1076,6 +1093,108 @@ def delete_lambda_function(function_name: str) -> Dict[None, None]:
     return {}
 
 
+def get_lambda_url_config(api_id, region=None):
+    lambda_backend = LambdaRegion.get(region)
+    url_configs = lambda_backend.url_configs.values()
+    lambda_url_configs = [config for config in url_configs if config.get("CustomId") == api_id]
+    return lambda_url_configs[0]
+
+
+def event_for_lambda_url(api_id, path, data, headers, method) -> dict:
+    rawPath = path.split("?")[0]
+    rawQueryString = path.split("?")[1] if len(path.split("?")) > 1 else ""
+    queryStringParameters = (
+        {} if not rawQueryString else dict(urllib.parse.parse_qsl(rawQueryString))
+    )
+
+    requestContext = {
+        "acountId": "anonymous",
+        "apiId": api_id,
+        "domainName": headers.get("Host", ""),
+        "domainPrefix": api_id,
+        "http": {
+            "method": method,
+            "path": rawPath,
+            "protocol": "HTTP/1.1",
+            "sourceIp": headers.get("Remote-Addr", ""),
+            "userAgent": headers.get("User-Agent", ""),
+        },
+        "requestId": short_uid(),
+        "routeKey": "$default",
+        "stage": "$default",
+        "time": datetime.utcnow().isoformat(),
```

Comment:
The example value here: https://docs.aws.amazon.com/lambda/latest/dg/urls-invocation.html does not look like iso format. Can you check against AWS if the format is indeed like in the docs, and adapt this?